### ISSUE TITLE
Recommended channel

### ISSUE THREAD

**Issue by nonneentered** [yuliskov/SmartTubeNext#1655]
STN Beta Version 15.06 I dont know if this is related to the Nvidia shield or STN, However all other apps work as expected. Under Customize your home screen/customize channels/STN Beta/STN Beta channels the Recommended channel just says "Channel is empty". Well it isnt obviously. Turning the switch on and off does nothing if i open the app there are videos showing up in recommended . If i go to the youtube app and switch on recommended the youtube one works fine. Tried rebooting tried unplugging for a few minutes tried clearing the cache . Thing is under play next i have a lot of videos and i would prefer not to lose them if i can help it so i have not cleared data . So is this a shield thing or ?

**Comment by pda1972** [comment]
STN Beta Version 15.06 on 29/06/22 didn't work, issues on Android TV launcher (on Nvidia Shield), Subscriptions Channel fine, Recommended channel empty, History channel not updating. 

These Issues were on 29/06/22 Morning. 

By about 18.00GMT All 3 Channels working, Subscriptions, Recommended channel populated, History channel updating, 15.06 Beta App itself working 100%, issues were with Channels, which seemed to be all OK now.

Morning of 30/06/22, notified of update available to 15.08, installed update.

15.08 Beta App appears to working well, issue is again with channels, but worse. Subscription Channel fine, Recommended channel empty again, but now History channel is not showing at all (as opposed to showing but not updating). This is all under the customize channels option on the Shield.

Have rebooted shield several times, uninstalled 15.08 Beta, reinstalled 15.08 Beta, Rebooted Shield again, still same problem with Channels.

Plex and Emby channels are all working, so are Kodi Channels (just so you are aware)

Thanks

**Comment by MRobi1** [comment]
Same issue here on 14.99 stable on multiple NVidia Shield devices. Saw a message pop up on one of them saying it was removing the channel since it was empty. I can go into customize channels and it's turned on but right under it says "Channel is empty"

**Comment by nonneentered** [comment]
Yes same here there was an update last night to 15.08 I think that seemed
to have a fix for the issue but I still have it so hopefully it will get
fixed soon.

**Comment by pda1972** [comment]
Tried to install Stable 14.99, same issues, Subscription channel fine, Recommended Channel empty, History channel not showing at all. I have not done any system updates on my Nvidia Shield, all was working 100% (both apps and channels) until about 28/6/22. For Reference, other app channels (Plex, Kodi etc) are still fine. Thanks

**Comment by nonneentered** [comment]
Similar to mine. It stopped working at the same date as yours however my
shield is fully updated that would be the only difference.

**Comment by pda1972** [comment]
All appears to be OK now on my Shield (20.00 GMT) All channels showing and being updated, BETA 15.09 update seems to have done the trick. Thank you for your efforts.

**Comment by nonneentered** [comment]
Yep same here I had to check for the update but all fixed now thanks for
the quick fix

**Comment by nonneentered** [comment]
Beta 15.09 fixed the issue thanks.

**Comment by MRobi1** [comment]
Is there a simple path to switch to the beta channel from within the app or
does it need to be side loaded in place of the stable version?
